src/main/java/cube/model/food/Food.java

```diff
@@ -113,16 +116,24 @@ public Date getExpiryDate() {
 	 *
 	 * @param newRevenue New total revenue made from selling the product.
 	 */
+	/*
 	public static void updateRevenue(double newRevenue) {
-		revenue = newRevenue;
+			revenue = newRevenue;
 	}
+    */
 
 	/**
-	 * Gets the revenue earned from selling the product.
+	 * Generates the total revenue earned from selling the product.
 	 *
-	 * @return the revenue earned from selling the product.
+	 * @return the total revenue earned from selling the product.
 	 */
 	public static double getRevenue() {
```

Comment:
Should we consider to store the variable somewhere for better access times?

Reply:
Hi Ian,

Thank you for reaching out and asking the question. Please allow me to answer.

Static variable for revenue is removed because it does not make sense to keep a static variable. Surely it will have a better access time, but it flies in the face of the purpose of the code.

In the context here, when referring to the profits and revenue, we don't want it to refer to everything under the sun - the business people will not care about how much they have made ever since the first day they started business, but rather, more concerned with how much they have made in a given time frame - e.g. how much they have made in the past few months, or the first quarter of a year, per se.

Hope that clarifies. I would be happy to answer any further questions that you may have.

P.S. You are doing an excellent job with the GUI component. Keep it up! :)

Pengfei